---
float32 hardIronX
float32 hardIronY
float32 hardIronZ